exasol_sagemaker_extension/autopilot_utils/enpoint_deployment.py

```diff
@@ -11,7 +11,7 @@ def deploy(job_name: str, endpoint_name: str,
 
         unique_endpoint_name = endpoint_name
         automl = AutoML.attach(auto_ml_job_name=job_name)
-        best_candidate = automl.describe_auto_ml_job(job_name)["BestCandidate"]
+        best_candidate = automl.best_candidate()
```

Comment:
if a job is interrupted before completion due to working of one of the "max_runtime" stopping criteria, autopilot might have several candidate but might not have an best candidate. we can throw an exception saying that there is not a best candidate. what do you think?